src/libraries/TransientStorage.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.26;
+
+/// @title  TransientStorage
+library TransientStorage {
+    function tstore(bytes32 slot, address value) internal {
```

Comment:
why do we have to name `tstore` and `tload`?

Reply:
I think it is valuable to name this explicitly. Also what Uniswap/Balancer/OZ are all doing.